modules/connectors/canadapost/karrio/providers/canadapost/shipment/create.py

```diff
@@ -89,8 +90,8 @@ def shipment_request(
     label_encoding, label_format = provider_units.LabelType.map(
         payload.label_type or "PDF_4x6"
     ).value
-    group_id = str(uuid.uuid4().hex)
-    customer_request_ids = [f"{group_id}" for _ in range(len(packages))]
+    group_id = lib.to_date(datetime.datetime.now(), "%Y%m%d")
+    customer_request_ids = [f"{group_id}{_}" for _ in range(len(packages))]
```

Comment:
```suggestion
    customer_request_id = str(uuid.uuid4().hex)
    customer_request_ids = [f"{customer_request_id}{_}" for _ in range(len(packages))]
```
if `group_id` is not unique then we need to generate a special uuid for it

Reply:
True forgot to update this one.